src/libs/Navigation/OnyxTabNavigator.tsx

```diff
@@ -1,12 +1,17 @@
 import {createMaterialTopTabNavigator, MaterialTopTabNavigationEventMap} from '@react-navigation/material-top-tabs';
 import {EventMapCore, NavigationState, ScreenListeners} from '@react-navigation/native';
 import React from 'react';
+import {Platform} from 'react-native';
 import {withOnyx} from 'react-native-onyx';
 import {OnyxEntry} from 'react-native-onyx/lib/types';
 import Tab from '@userActions/Tab';
 import ONYXKEYS from '@src/ONYXKEYS';
 import ChildrenProps from '@src/types/utils/ChildrenProps';
 
+const TabNavigatorAnimationEnabled = ['ios', 'android'].includes(Platform.OS);
```

Comment:
Also should this be camelCase?

Reply:
Good catch, thanks! I will rename it when resolving https://github.com/Expensify/App/pull/32577#discussion_r1431917738